### PR TITLE
Crafting Precurser

### PR DESCRIPTION
**Pull request by PlasticCat**
## Description
added craftable parts, gave them something to craft with them, more to come

Note: wasteland soap is intentionally slow and expensive to make it less valuable than a normal cleaner and helps reduce soap memes.

## Motivation and Context
Precursor work for the creation of more complex crafts that require working together

## How Has This Been Tested?
crafted the crafts

## Changelog (necessary)
:cl:
add: wasteland soap and recipe
add: large and small gear recipes
add: hide glue recipe, turpentine recipe, wonder glue recipe
imageadd: pemmican sprite, hide glue sprite

/:cl:

### CODE REVIEW

**Comment by Laus4Deus** [comment]
I am looking at the files changed, but I need a precise description on the changes

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1081,6 +1081,31 @@
...
+	icon_state = "dread_ipad"
+	slot_flags = ITEM_SLOT_BELT
+	w_class = WEIGHT_CLASS_SMALL
+	var/cooldown = 0
+
+/obj/item/toy/clockwork_watch/attack_self(mob/user)
+	if (cooldown < world.time)
+		cooldown = world.time + 1800 //3 minutes
+		user.visible_message("<span class='warning'>[user] rotates a cogwheel on [src].</span>", "<span class='notice'>You rotate a cogwheel on [src], it plays a loud noise!</span>", "<span class='italics'>You hear cogwheels turning.</span>")
+		playsound(src, 'sound/magic/clockwork/ark_activation.ogg', 50, 0)
```
Comment (Laus4Deus) [inline code/game/objects/items/toys.dm:1101]: ark_activation? Could you please find another sound for it

Comment (PlasticCat) [reply code/game/objects/items/toys.dm]: used gun slide lock 1 to hopefully give the mechanical clicking sound better.

**Comment by PlasticCat** [comment]
Better description added

**Comment by judgex** [comment]
missing file sound/weapon/clockwork/gun_slide_lock_l.ogg

**Pull request merged**